src/utils/createEndpoint.js

```diff
@@ -29,6 +29,9 @@ function createSnippet(data) {
   if (data.request?.headers) {
     harObject.headers = Object.entries(data.request.headers).map(([name, value]) => ({ name, value }));
   }
+  if (data.request?.queryString) {
```

Comment:
Added tests for this in an earlier PR: https://github.com/centrapay/centrapay-docs/pull/899/files#diff-ea1386bdb2d89a1d4246ff57b6c561163fde125e9671602938574991670e20b1